UPGRADE-2.8.md

```diff
@@ -475,6 +475,8 @@ Security
    }
    ```
 
+ * The `AbstractVoter` is deprecated. Upgrade to `Voter` instead.
+
```

Comment:
If people were using AbstractVoter in 2.7, they can read how to upgrade to 2.8;
Then they could upgrade with the new Voter. Like that it's done step by step.

But I can merge all steps together.